.eslintignore

```diff
@@ -1,3 +1,3 @@
 public/lib/*
 public/js/*
-dist
```

Comment:
dist files are automatically transpiled (files under `/view/atlasmaker/dist` and `/view/brainbox/dist`), it makes no sense to try to lint them

Reply:
Ah no!
This crept through due to some error.
It wasn't meant to be here..

Reply:
Hum, I'm not sure what do you mean...
It's me who added this because you removed view from eslint ignore, which is not a bad idea because there are also source files in the view folder, but we still have to eslint ignore dist folders.